### PR TITLE
Homepage - Hero

### LINKED ISSUE #72: Hero Section - Homepage
Homepage-hero
Implement the new design as per the Figma designs. 

## Acceptance criteria
The hero page matches the Figma designs
Design sign off
The hero section is accessible
KPIs must be as good as, or better, than existing KPIs
## Considerations
Google Analytics must still work as before

### PR DESCRIPTION
**Pull request by bp289**
closes #72 
Changes made:
Fonts.css classes changed to suitable names. 
New CSS file for buttons.

Html lint changes:
since we are using kebab case classes for CSS, the "id-class-style" has been switched to "dash" which shouldn't give linting errors now.

### CODE REVIEW

File: assets/style/buttons.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  background: var(--primary-p400);
+  transition: 0.3s;
+  color: #fff;
+  border: none;
+  font-family: "Atkinson Hyperlegible";
+}
+button:hover {
+  background: var(--primary-p500);
+}
+.hero-button {
```
Comment (MariamM110) [inline assets/style/buttons.css]: Most of the css here matches the h6 in fonts.css, would it be easier to add another class for hero-button in font css ?

File: assets/style/buttons.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.button {
+  padding: 0.6rem 1.625rem;
+  border-radius: 8rem;
+  background: var(--primary-p400);
+  transition: 0.3s;
+  color: #fff;
+  font-family: inherit;
```
Comment (SociableSteve) [inline assets/style/buttons.css]: Do you need inherit?

File: assets/style/buttons.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  margin-bottom: 0.3rem;
+}
+
+.button-hero img {
+  width: 1rem;
+  height: 1rem;
+  vertical-align: middle;
+}
+
+/* stylelint-disable */
```
Comment (SociableSteve) [inline assets/style/buttons.css]: Why are you disabling linting?

File: assets/style/buttons.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+.button:hover {
+  background: var(--primary-p500);
+}
+
+.button-hero {
+  padding: 1rem 1.625rem;
+  border-radius: 16rem;
+  background: var(--gradient-brandreverse);
+  margin: auto;
```
Comment (SociableSteve) [inline assets/style/buttons.css]: prefer `margin: auto auto 0.3rem;` to keep it all on one line

File: assets/style/buttons.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
+.button {
+  padding: 0.6rem 1.625rem;
+  border-radius: 8rem;
+  background: var(--primary-p400);
+  transition: 0.3s;
```
Comment (SociableSteve) [inline assets/style/buttons.css:6]: We should consider `prefers-reduced-motion` for some CSS things, like animations. //cc @Mariam-mohamed110

File: assets/style/fonts.css [hunk 6]
```diff
@@ -21,79 +21,99 @@ h2,
 h3,
 h4,
 h5,
-h6 {
+h6,
+.text-h1,
```
Comment (SociableSteve) [inline assets/style/fonts.css:25]: The `.text-h*` classes don't appear to exist. They really shouldn't either - we should by using the relevant h tags.

File: assets/style/fonts.css [hunk 7]
```diff
@@ -21,79 +21,99 @@ h2,
...
-a {
+a,
+.base {
   margin-top: 0;
   margin-bottom: 0;
   font-size: 1rem; /* 16px */
   line-height: 130%;
   letter-spacing: 0;
   text-decoration: none;
+  color: #070707;
```
Comment (SociableSteve) [inline assets/style/fonts.css]: If this is meant to apply to the entire body content as a default, put it into the `body` block

File: assets/style/hero.css [hunk 8]
```diff
@@ -0,0 +1,29 @@
+.hero {
```
Comment (SociableSteve) [inline assets/style/hero.css:1]: We have a `.container` class for constraining the width. Prefer that so it's in one place to change if we ever decide to.

File: assets/style/hero.css [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+  max-width: 1400px;
+  margin: 72px auto;
```
Comment (SociableSteve) [inline assets/style/hero.css]: do we always want 72px, or should it be 4.5rem?

File: assets/style/hero.css [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+/* stylelint-enable */
+
+.hero p {
+  margin-top: 24px;
+  margin-bottom: 40px;
+}
+
+@media screen and (width <= 768px) {
+  .hero p {
+    font-size: 1rem;
```
Comment (SociableSteve) [inline assets/style/hero.css:26]: Did it ever change from 1rem?

**Review by SociableSteve — CHANGES_REQUESTED** [review]

File: assets/style/fonts.css (hunk 6 above)
Comment (bp289) [reply assets/style/fonts.css:25]: I was using this or the button on the hero as it matched the styling for the h6, I was going to originally use a h6 but  Isnt it an accesibility issue if I use a `h6` before a `h3` in the dom?

File: assets/style/hero.css (hunk 10 above)
Comment (bp289) [reply assets/style/hero.css:26]: The hero should have the `text-large` class applied so it should be 1.25 rem to start off with.

File: assets/style/buttons.css (hunk 3 above)
Comment (bp289) [reply assets/style/buttons.css]: sorry forgot to change this back, updated now!

**Review by MariamM110 — APPROVED** [review]

**Pull request merged**